source/Halibut/HalibutRuntime.cs

```diff
@@ -317,6 +317,26 @@ public async Task DisconnectAsync(ServiceEndPoint endpoint, CancellationToken ca
             await connectionManager.DisconnectAsync(endpoint, log, cancellationToken);
         }
 
+        [Obsolete("Use DisposeAsync")]
+        public void Dispose()
+        {
+            pollingClients.Dispose();
+            connectionManager.DisposeAsync().GetAwaiter().GetResult();
```

Comment:
Since we're doing `GetAwaiter().GetResult()` here anyway, did we want to make `Dispose` simply call `DisposeAwait.GetAwaiter().GetResult()` instead?